### DISCUSSION TITLE
.NET MAUI Android how to transcode the video resolution to 720p to reduce the upload time to server

### DISCUSSION THREAD

**Discussion by SamantaSharma** [dotnet/maui#20459] [Q&A]
I am researching ways to reduce upload time for videos to server in .NET MAUI Android. Without transcoding the video, it takes more than 30 seconds - 1 minute to upload the video. With IOS, I have used AVAsset and it works it's magic. However for Android, we used Xamarin.Android.MP4Transcoder in xamarin, however there are no library as such available for .NET Maui. Has anyone successfully implemented it on .NET Maui?

**Comment by giuseppenovielli** [comment]
Hi,
this library: https://github.com/giuseppenovielli/Xam.Android.LiTr

is built fo Xamarin.Android, but you can try to:
1)create new binding project for .NET MAUI Android: https://learn.microsoft.com/en-us/dotnet/maui/migration/android-binding-projects?view=net-maui-8.0
2)add .aar https://mvnrepository.com/artifact/com.linkedin.android.litr/litr/1.5.5
3)Compile

**Comment by MDhifi** [comment]
did you find a solution?
